src/move/econia/Move.toml

```diff
@@ -1,7 +1,8 @@
 [package]
 name = "Econia"
-version = "4.1.0"             # Per SemVer.
+version = "4.1.0"                                     # Per SemVer.
 upgrade_policy = "compatible"
+authors = ["Econia Labs (developers@econialabs.com)"]
```

Comment:
spelled right and everything
![giphy (2)](https://github.com/econia-labs/econia/assets/8174594/d8a39a9e-96a9-437e-96b4-ddd06780ffc4)